### ISSUE TITLE
Problem with fetching balance

### ISSUE THREAD

**Issue by ThePatrykOOO** [QuilibriumNetwork/ceremonyclient#78]
I have problem with fetching balance. My docker is running properly. Look at below. Repo is up to date

```
~/ceremonyclient# docker-compose exec node go run ./... -balance
panic: error getting token info: rpc error: code = Unknown desc = get token info: get highest candidate data clock frame: item not found

goroutine 1 [running]:
main.main()
	/opt/ceremonyclient/node/main.go:101 +0x7f4
exit status 2

```

**Comment by sercanCyberVision** [comment]
Please see this https://github.com/QuilibriumNetwork/ceremonyclient/issues/54.
It takes some time for your node to be synced.

**Comment by ThePatrykOOO** [comment]
```
~/ceremonyclient# nc -zv 0.0.0.0 8337
Connection to 0.0.0.0 8337 port [tcp/*] succeeded!
root@quadlirium-2:~/ceremonyclient# docker-compose exec node go run ./... -balance
panic: error getting token info: rpc error: code = Unknown desc = get token info: get highest candidate data clock frame: item not found

goroutine 1 [running]:
main.main()
	/opt/ceremonyclient/node/main.go:101 +0x7f4
exit status 2
```

**Comment by sercanCyberVision** [comment]
@ThePatrykOOO, based on my limited experience, you are good. Your node is connected, it will take 4-5 hours (depends on network) for node to be synced. After that you will see your balance as zero.

**Comment by mscurtescu** [comment]
If you call GetNodeInfo over gRPC and the response has no `maxFrame` then it seems that the node is still in an early state. Obviously it should not crash, this is a bug.

Calling GetTokenInfo also fails in this state.

This is how you can make these gRPC calls:
https://github.com/mscurtescu/ceremonyclient/wiki/gRPCurl-How-To#getnodeinfo

**Comment by cratharavi** [comment]
Hello, am getting this err 
panic: error getting token info: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp 127.0.0.1:8337: connect: connection refused"

goroutine 1 [running]:
main.printBalance(0xc0003ea309?)
        /home/roxstacr/q/ceremonyclient/node/main.go:527 +0x259
main.main()
        /home/roxstacr/q/ceremonyclient/node/main.go:126 +0x7ff
exit status 2

**Comment by mscurtescu** [comment]
> Hello, am getting this err panic: error getting token info: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp 127.0.0.1:8337: connect: connection refused"
> 
> goroutine 1 [running]: main.printBalance(0xc0003ea309?) /home/roxstacr/q/ceremonyclient/node/main.go:527 +0x259 main.main() /home/roxstacr/q/ceremonyclient/node/main.go:126 +0x7ff exit status 2

Is your gRPC port configured? Is the node actually running?

How did you install the node, following what instructions? On what type of server? What OS?

What command did you try to run when you got this error?

**Comment by sleeply7777** [comment]
****/exe/node just listened 8336 after I run `sh init.sh && GOEXPERIMENT=arenas go run ./...`
no process listened 8337 -> gRPC was not running?
ubuntu 22.04
The startup log output is as follows:
```
{"level":"info","ts":1716532677.4011705,"caller":"master/master_clock_consensus_engine.go:229","msg":"broadcasting self-test info"}
{"level":"info","ts":1716532677.6084948,"caller":"master/master_clock_consensus_engine.go:215","msg":"peers in store","peer_store_count":96,"network_peer_count":67}
{"level":"info","ts":1716532687.3644497,"caller":"p2p/blossomsub.go:348","msg":"connecting to bootstrap","peer_id":"QmSN9RPRSwK54GwB48SkSsyWFEDTnixxuzXBkaor25J6kV"}
{"level":"info","ts":1716532687.609508,"caller":"master/master_clock_consensus_engine.go:215","msg":"peers in store","peer_store_count":96,"network_peer_count":67}
```

**Comment by agostbiro** [comment]
@sleeply7777 could you elaborate on the issue that you're facing? I'm not sure I understand your message

**Comment by sleeply7777** [comment]
After running `GOEXPERIMENT=arenas go run ./...` and waiting for some time with continuous logging, such as `{"level":"info","ts":1716780783.9985313,"caller":"node/main.go:422","msg":"generating 65536 degree proof metric"}`, I opened a new terminal and ran `GOEXPERIMENT=arenas go run ./... -balance`, but received an error message: `"transport: Error while dialing: dial tcp 127.0.0.1:8337: connect: connection refused"`. I checked and confirmed that there is no process occupying port 8337.

Is the Quilibrium Node running normally?
How can I check the balance?
